src/libultra/gu/rotate.c

```diff
@@ -1,7 +1,7 @@
 #include "global.h"
 
 void guRotateF(f32 m[4][4], f32 a, f32 x, f32 y, f32 z) {
-    static f32 D_80134D10 = M_PI / 180.0f;
+    static f32 dtor = M_PI / 180.0f;
```

Comment:
A slight nit, while I can see this is intended to be degrees to radians, I think "degtorad" or something of the sort would be clearer. I see dtor and think "destructor".

Reply:
Im assuming with a name like this its probably from mdebug info of some kind, but @Thar0 should confirm

Reply:
Yeah it's from libultra debug info

Reply:
Sounds good, thanks for the clarification!